content/docs/next-release/operator.md

```diff
@@ -0,0 +1,666 @@
+
+# Installing the Operator
+
+{{< info >}}
+The Jaeger Operator version tracks one version of the Jaeger components (Query, Collector, Agent). When a new version of the Jaeger components is released, a new version of the operator will be released that understands how running instances of the previous version can be upgraded to the new version.
+{{< /info >}}
+
+## Installing the Operator on Kubernetes
+
+The following instructions will create the `observability` namespace and install the Jaeger Operator.
+
+{{< info >}}
+Make sure your `kubectl` command is properly configured to talk to a valid Kubernetes cluster. If you don't have a cluster, you can create one locally using [`minikube`](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+{{< /info >}}
+
+To install the operator, run:
+<!--TODO - Does Kubernetes have privileged users? Needs to be run as a system:admin on OKD/OpenShift.-->
+
+```bash
+kubectl create namespace observability # <1>
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing_v1_jaeger_crd.yaml # <2>
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml
+```
+<1> This creates the namespace used by default in the deployment files. If you want to install the Jaeger operator in a different namespace, you must edit the deployment files to change `observability` to the desired namespace value.
+
+<2> This installs the "Custom Resource Definition" for the `apiVersion: jaegertracing.io/v1`
+
+At this point, there should be a `jaeger-operator` deployment available.  You can view it by running the following command:
+
+```bash
+$ kubectl get deployment jaeger-operator -n observability
+
+NAME              DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+jaeger-operator   1         1         1            1           48s
+```
+
+The operator is now ready to create Jaeger instances.
+
+## Installing the Operator on OKD/OpenShift
+
+<!-- TODO: Add instructions for installing via the operatorhub? -->
+
+The instructions from the previous section also work for installing the operator on OKD or OpenShift. Make sure you are logged in as a privileged user, such as `system:admin`, when you install the role based acces control (RBAC) rules, the custom resource definition, and the operator.
```

Comment:
At least for CRC, the privileged user is now "kubeadmin". Not sure there's even a "system:admin" anymore.